### ISSUE TITLE
Missing value for property 'subscriptions.name'

### ISSUE THREAD

**Issue by irepan** [RocketChat/Rocket.Chat.ReactNative#385]
- Your Rocket.Chat server version: 0.66.2

- Device model (or emulator) you're running with: ####

For android: Xiaomi redmiS3 - 6.0.1
- Steps to reproduce:
The App stops working after login, we open it again and again and just breaks, but if we try to login with a different user, it works fine, so far it has happened to 2 different users.

Stack trace
```
Fatal Exception: com.facebook.react.common.JavascriptException: TaskQueue: Error with task : Missing value for property 'subscriptions.name', stack:
<unknown>@1284:1735
<unknown>@1284:1707
<unknown>@1284:1686
value@513:1025
b@512:1092
<unknown>@349:1670
w@349:643
x@349:890
callImmediates@349:3071
value@344:3183
<unknown>@344:1516
value@344:3009
value@344:1486
value@344:1421

       at com.facebook.react.modules.core.ExceptionsManagerModule.showOrThrowError(ExceptionsManagerModule.java:54)
       at com.facebook.react.modules.core.ExceptionsManagerModule.reportFatalException(ExceptionsManagerModule.java:38)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.facebook.react.bridge.JavaMethodWrapper.invoke(JavaMethodWrapper.java:372)
       at com.facebook.react.bridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:160)
       at com.facebook.react.bridge.queue.NativeRunnable.run(NativeRunnable.java)
       at android.os.Handler.handleCallback(Handler.java:742)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:29)
       at android.os.Looper.loop(Looper.java:157)
       at com.facebook.react.bridge.queue.MessageQueueThreadImpl$3.run(MessageQueueThreadImpl.java:192)
       at java.lang.Thread.run(Thread.java:818)
```

**Comment by diegolmello** [comment]
Hi, @irepan!
This is happening because name is required on `realm.js`, but it was fixed on #375.
Are you sure you are up-to-date?

Can you find what is the room object realm is failing to add?

**Comment by irepan** [comment]
Yes, this was fixed, thank you
